### DISCUSSION TITLE
ESP8266 with 0.96" OLED

### DISCUSSION THREAD

**Discussion by Gohtraw** [VeryKross/BGBuddy#6] [Show and tell]
Hi, I got it to run on this board by adding "  Wire.begin(14,12);" in line 101 in the setup routine.
![ESP8266 OLED](https://github.com/VeryKross/BGBuddy/assets/112734607/7912e8dd-26e0-4e5e-9882-f916fbe05672)

**Comment by VeryKross** [comment]
Great to hear! That's something I'm seeing too - different manufacturers of this module sometimes swap the pins on the OLED SDA and OLED SDL. I've added two new binaries in the repo for this module, each one with either "DA12" or "DA14" depending on which your module might be setup as.
